liwords-ui/src/store/store.tsx

```diff
@@ -756,6 +763,20 @@ export const Store = ({ children, ...props }: Props) => {
   return <React.Fragment children={ret} />;
 };
 
+const ResetStoreContext = createContext({ resetStore: defaultFunction });
+export const useResetStoreContext = () => useContext(ResetStoreContext);
+
+export const Store = ({ children }: { children: React.ReactNode }) => {
+  // In JS the | 0 loops within int32 and avoids reaching Number.MAX_SAFE_INTEGER.
+  const [storeId, resetStore] = React.useReducer((n) => (n + 1) | 0, 0);
+
+  return (
+    <ResetStoreContext.Provider value={{ resetStore }}>
+      <RealStore key={storeId} children={children} />
```

Comment:
i really didn't know this could be done -- it essentially just creates an entire new store?